packages/@react-spectrum/inlinealert/src/InLineAlert.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import AlertMedium from '@spectrum-icons/ui/AlertMedium';
+import {classNames, SlotProvider, useDOMRef, useStyleProps} from '@react-spectrum/utils';
+import {DOMRef} from '@react-types/shared';
+import {filterDOMProps} from '@react-aria/utils';
+import {Grid} from '@react-spectrum/layout';
+import InfoMedium from '@spectrum-icons/ui/InfoMedium';
+// @ts-ignore
+import intlMessages from '../intl/*.json';
+import React from 'react';
+import {SpectrumInLineAlertProps} from '@react-types/inlinealert';
+import styles from '@adobe/spectrum-css-temp/components/inlinealert/vars.css';
+import SuccessMedium from '@spectrum-icons/ui/SuccessMedium';
+import {useLocalizedStringFormatter} from '@react-aria/i18n';
+import {useProviderProps} from '@react-spectrum/provider';
+
+let ICONS = {
+  info: InfoMedium,
+  positive: SuccessMedium,
+  notice: AlertMedium,
+  negative: AlertMedium
+};
+
+function InLineAlert(props: SpectrumInLineAlertProps, ref: DOMRef<HTMLDivElement>) {
```

Comment:
I orignally had `InlineAlert` but then noticed `spectrum-css` uses `InLineAlert` since Spectrum calls this component `In-Line Alert` and not `Inline Alert`. Anyway, I'm fine with either but went with consistency.

https://github.com/adobe/spectrum-css/blob/main/components/inlinealert/index.css#L13

Reply:
After talking with the team, it looks like `InlineAlert` is preferable for now. Thanks!

Reply:
Does this include renaming ALL the spectrum-css class names or just the React names?

Reply:
can leave the classnames alone, they should just match whatever spectrum-css has